psvpfsparser/PsvPfsParserConfig.cpp

```diff
@@ -100,11 +100,6 @@ int parse_options(int argc, char* argv[], PsvPfsParserConfig& cfg)
          f00d_cache = vm[F00D_CACHE_NAME].as<std::string>();
       }
 
-      if(f00d_url.empty() && f00d_cache.empty())
```

Comment:
Can you explain this change. Why did you remove this check?

Reply:
If both url and cache options are not specified, then F00DNativeKeyEncryptor will be used. So the program should not terminate here.

Reply:
I see. That is correct.